css/amp.css

```diff
@@ -47,6 +47,39 @@ body {
   text-size-adjust: 100%;
 }
 
+
+/**
+ * iOS-Embed Wrapper mode. The `body` is wrapper into `i-amp-html-wrapper`
+ * element.
+ */
+html.-amp-ios-embed {
+  overflow-y: auto !important;
+  -webkit-overflow-scrolling: touch !important;
+}
+
+/** Wrapper for iOS-Embed-Wrapper mode. */
+i-amp-html-wrapper {
+  overflow-x: hidden !important;
+  overflow-y: auto !important;
+  -webkit-overflow-scrolling: touch !important;
+  position: absolute !important;
+  top: 0 !important;
+  left: 0 !important;
+  right: 0 !important;
+  bottom: 0 !important;
+  margin: 0 !important;
+  display: block !important;
+}
+
+i-amp-html-wrapper > body {
+  position: relative !important;
```

Comment:
So that `position:absolute` elements are positioned relative to the `body` and not `i-amp-html-wrapper`.